modules/stanford_profile_styles/package.json

```diff
@@ -14,7 +14,7 @@
     "@babel/preset-env": "^7.22.4",
     "babel-loader": "^9.1.2",
     "css-loader": "^6.8.1",
-    "decanter": "^6.2.14",
+    "decanter": "^6.3.2",
```

Comment:
Please commit the updated yarn.lock file.

Reply:
Also there are about 9 places decanter is used in this repo. I'm not sure if they all need updates

Reply:
Ok. Thank you. I will

Reply:
I updated Decanter in this repo and in Stanford Basic.